### PR TITLE
Merge ML sprint2 

### PR DESCRIPTION
**Pull request by maxwang2004**
# Description

Adds: clip_test.py. Input to classify_img: pillow image. Outputs: a string label describing the most likely primary food item in the image. 
Requires: clip to be installed using requirements.txt

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Ran on cheeseburger.jpeg and fries.jpg

### CODE REVIEW

File: ml/clip_test.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    with torch.no_grad():
+        image_features = model.encode_image(image)
+        text_features = model.encode_text(text)
+        
+        logits_per_image, logits_per_text = model(image, text)
+        probs = logits_per_image.softmax(dim=-1).cpu().numpy()
+
+
+    print(labels[np.argmax(probs)])
```
Comment (bryantpark04) [inline ml/clip_test.py]: this should be return, not print

**Review by bryantpark04 — CHANGES_REQUESTED** [review]
